File: youlexuan_parent/youlexuan_sellergoods_interface/src/main/java/com/offcn/service/BrandService.java
package com.offcn.service;

import com.offcn.entity.PageUtil;
import com.offcn.pojo.TbBrand;

import java.util.List;

public interface BrandService {

	public List<TbBrand> findAll();

	public void addBrand(TbBrand brand);

	public TbBrand findOne(Long id);

	public void updateBrand(TbBrand brand);

	public void deleteByIds(Long[] ids);

	public PageUtil<TbBrand> findPage(Integer thisPage, Integer pageSize, TbBrand brand);
}

File: youlexuan_parent/youlexuan_manager_web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
	this.findPage = function (thisPage, pageSize, entity) {
		return $http.post("/brand/findPage.do?thisPage=" + thisPage + "&pageSize=" + pageSize, entity);
	};

	this.updateInfo = function (info, entity) {
		return $http.post("/brand/" + info + ".do", entity);
	};

	this.delete = function (ids) {
		return $http.get("/brand/deleteByIds.do?ids=" + ids);
	};

	this.findOne = function (id) {
		return $http.get("/brand/findOne.do?id=" + id);
	};
});